ods_ci/utils/scripts/rosa/rosa.py

```diff
@@ -21,6 +23,22 @@ def __init__(self, args={}):
         self.rosa_version = args.get("rosa_version")
         self.channel_name = args.get("channel_name")
 
+    def set_rosa_version(self):
+        version_match = re.match(r"(\d+\.\d+)\-latest", self.rosa_version)
+        if version_match is None:
+            log.info(f"Using the rosa version given by user: {self.rosa_version}")
+            return
+        version = version_match.group(1)
+        channel_group = ""
+        if self.channel_name == "candidate":
+            channel_group = f"--channel-group {self.channel_name}"
```

Comment:
I don't fully understand this, TBH. Disclaimer: I have no knowledge about rosa CLI so I may be wrong.

TBH, I would rather include the `--channel-group` based on whether a `channel_name` is defined or not rather than including based on the type of the channel name here. Does that make sense? Because, with this implementation, it means that only in case that `candidate` channel name is used, this argument will be given to the command. What if this channel will be renamed in the future? What is the reason not to add this argument to the command when the channel name is different?

Reply:
you are right, we can add --channel-group  in any case 
Done!!